core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20220929_creating_landing_zones.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="dummy" author="yunip20" id="create_CREATING_LANDING_ZONES">
+        <createTable tableName="CREATING_LANDING_ZONES">
```

Comment:
As we discussed in refinement, I think we want to generalize this table so it is a workspace manager resource monitor. The primary key would then be the resource ID, the job control ID shouldn't mention landing zones specifically, and we want an entry for the resource type.